src/table/table.scss

```diff
@@ -350,12 +386,20 @@
   display: flex;
   flex-grow: 1;
   min-width: $iui-xxl;
-  min-height: $iui-baseline * 5;
+  min-height: $iui-baseline * 5 + 2px; // +2px for border
```

Comment:
This makes inner cell height to remain `55px` because previously it was `55px` and `2px` from borders on the row. It also makes header a bit bigger than it was before but now header row and body rows are the same height.

Reply:
Maybe I should revert header to 55px 😕
![image](https://user-images.githubusercontent.com/36186912/167783407-9b2a09ba-772e-473c-b882-fe027e263787.png)

Reply:
Reverted